### DISCUSSION TITLE
230704 1차 데모데이 발표자료 회의 (15:20 ~ 15:40)

### DISCUSSION THREAD

**Discussion by seokhwan-an** [woowacourse-teams/2023-shook#16] [회의]
## 회의 안건
발표 자료에 대해서 회의를 진행한다.

**Comment by Cyma-s** [comment]
## 1차 데모데이 발표 자료 목차

최대 10분

- 프로젝트 설명
	- 스플릿 매드무비 시간
- 팀의 문화/원칙
	- 원칙으로 중요한 것만 이야기하기 (세부보다는 원칙 위주)
- 사용자 스토리
	- 최우선 페르소나
	- 시나리오
- 2차 데모데이에서 구현할 핵심 기능 목록
	- 킬링파트 등록
	- 킬링파트 조회
- Git Branch 전략
	- Branch 종류
	- release 아직 없는 이유

## 자료 분배

- 프로젝트 설명: 스플릿
- 팀의 문화/원칙: 우코
	- 발표하기 좋게 만들어오기
- 사용자 스토리: 우코
- 2차 데모데이에서 구현할 핵심 기능 목록: 바론
- Git Branch 전략: 베로

**Comment by Cyma-s** [comment]
## 다음 회의까지

- 로고
- 메인 컬러
